src/main/java/edu/tamu/app/controller/ButtonManagementController.java

```diff
@@ -25,7 +25,7 @@ public class ButtonManagementController {
     @RequestMapping("/all")
     @PreAuthorize("hasRole('ADMIN')")
     public ApiResponse getAll() {
-        return new ApiResponse(SUCCESS,persistedButtonRepo.findAll(new Sort(Sort.Direction.ASC, "name")));
+        return new ApiResponse(SUCCESS, persistedButtonRepo.findAll(Sort.by("name")));
```

Comment:
Should we still be explicit about the sort order?

https://docs.spring.io/spring-data/commons/docs/current/api/org/springframework/data/domain/Sort.html#by-org.springframework.data.domain.Sort.Direction-java.lang.String...-

Reply:
Including the Sort Order - findAll(Sort.by(Sort.Direction.ASC, "name")));